README.md

```diff
@@ -29,17 +27,21 @@ We are looking for help from our community to complete the development of v2.0 o
 - Create Jira issues via Mattermost UI (Desktop App and browser only)
 - Attach Mattermost messages to Jira issues via Mattermost UI (Desktop App and browser only)
 - Preview Jira issues in Mattermost when a ticket is referenced
-- Send direct messages for Jira at-mentions and issue assignments
+- (May not be released) Send direct messages for Jira at-mentions and issue assignments
 - Slash commands for
   - `/jira connect` - Connect to a Jira project and subscribe to events.
   - `/jira disconnect` - Disconnect from a Jira project and subscribe from events.
   - `/jira subscribe` - Subscribe a Mattermost channel to receive notifications for above issue updates in a Jira project.
-  - `/jira instance` - Manage connected Jira instances. Must have System Admin role in Mattermost.
+  - (May not be released) `/jira instance [add/list/select]` - Manage connected Jira instances. Must have System Admin role in Mattermost. Primarily intended for debugging and development purposes.
```

Comment:
I'm aware that we may not release this :) But I'm just documenting what we have now + adding a note that it may not be released with 2.0